### DISCUSSION TITLE
旧文明战斗机图纸

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#2503] [其他]
![image](https://i0.hdslb.com/bfs/article/4cbd77354be07d9347fd46145640d7e3cb7aebd7.jpg@1256w_944h_!web-article-pic.avif)
空洞里翻出来的 奇迹般的没有被侵蚀
有没有懂行的大佬看看这份图纸

**Comment by GrunderIndustries** [comment]
另外 有没有更多类似的图纸 有的话100菲林收一个

**Comment by ghost-feng** [comment]
有图纸又怎样，造出来才nb
